SU-HSDO/suhumsci
# STN-814:   Incorrect hover state on external link in the secondary menu on Colorful

# READY FOR REVIEW

## Summary
Bug: External links in the secondary menu are displaying the inline link hover state color of `tertiary-darken-20` on hover. The work in this PR updates them to global `black` on hover to match the hover state of other links within the menu.

## Need Review By (Date)
EOD 7/26

## Urgency
high

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests continue to pass.
2. To test the work in this PR you will need to add an external link to a secondary menu. Go to [Structure > Menus > Main Navigation](http://sparkbox-sandbox.suhumsci.loc/admin/structure/menu/manage/main?destination=/admin/structure/menu).
3. Click on the `Add Link` button.
    * Enter a Menu Link Title
    * Under `Link` add an external link such as https://google.com
    * Under `Parent Link` selected a page under <Main Navigation> such as `About`, `Courses`, or `Events` and click Save.
4. Go to the page that you added the external link. 
    - [x] Verify the external link color is black on hover. 
    - [x] Check Colorful & Traditional themes. It should be black on both themes.
    - [x] Turn on the dark color inversion theme setting. The text color should also be black on hover.
